lua/lspconfig/quick_lint_js.lua

```diff
@@ -0,0 +1,32 @@
+local configs = require("lspconfig/configs")
+local util = require("lspconfig/util")
+
+configs.quick_lint_js = {
+  default_config = {
+    cmd = {"quick-lint-js", "--lsp-server"},
+    filetypes = {"javascript"},
```

Comment:
Add `javascriptreact` which is a common filetype for `.jsx` files. Does it work for TypeScript projects as well?

```suggestion
    filetypes = {"javascript", "javascriptreact"},
```